src/app/views/company/settings/organization-settings.directive.coffee

```diff
@@ -52,6 +53,16 @@ DashboardOrganizationSettingsCtrl = ($scope, $window, MnoeOrganizations, Utiliti
     f = $scope.forms
     $scope.isChanged() && f.settings.$valid
 
+  $scope.isCurrencyChangeShown = ->
+    if MnoeOrganizations.role.isSuperAdmin()
+      $scope.availableBillingCurrencies()
```

Comment:
What the advantage of this vs:
```coffescript
$scope.isCurrencyChangeShown = -> MnoeOrganizations.role.isSuperAdmin()
$scope.currencies = MnoeConfig.availableBillingCurrencies()
```
?

Reply:
I added it like this as a form of secondary validation check to never load currencies if the user is not Super Admin.

Reply:
Ok fair enough. Maybe rename it to `$scope.getAvailableBillingCurrencies()` to make it more explicit?